### ISSUE TITLE
[FE] SPIKE - Full Authentication Flow

### ISSUE THREAD

**Issue by julieadair1** [department-of-veterans-affairs/va.gov-team#36746]
## How to configure this issue

- [ ] Attached to an Epic (what body of work is this a part of?)
- [ ] Correctly Labeled
- [ ] Fill in User Stories
- [ ] Fill in Documentation
- [ ] Fill in Tasks
- [ ] Fill in Acceptance Criteria

---

## User Story
As a veteran stepping through the pre-check-in process, after I have fully authenticated I shouldn't have to authenticate after I log in again and I should be returned to the page I intended to edit. 

- (%FILL IN%)
- (%FILL IN%)

## Related Documentation

> these should be links to feature docs

- [Sketch](https://www.sketch.com/s/e79a827e-42cf-4a82-b554-874c75b5c70e)
- [Product Docs](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/health-care/checkin)
- [Feature Doc](%FILL IN%)

## Description

**Note time boxing this spike to a 5 ~ a week(ish)

## Tasks

> this is a list of technical tasks that need for this ticket

- [ ] %Add any tasks%
- [ ] Create or update unit tests
- [ ] Create or update e2e tests
- [ ] Update relevant documentations
  - [ ] README
  - [ ] Use Cases

## Acceptance Criteria
1. Flow diagram 
2. Working prototype
3. Document testing strategy 

- [ ] %Add any additional Acceptance Criteria%
- [ ] Passes Accessibility checks using axe
- [ ] Deployed to staging.
- [ ] Designer has signed off on the feature
- [ ] Product team has signed off on the  feature

**Comment by brianseek** [comment]
After experimenting a bit with a local setup for vets-api, I discovered that the session for check-in gets overwritten by the session for full authentication. I met with Dillo and Kanchana and we discussed the issue, I updated an initial problem statement and a few questions in the doc that Dillo has setup: https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/checkin/engineering/architecture-docs/edit-demographics/demographics_edit_auth.md

At the moment, we don't have the ability to minimally auth and then full auth and come back to the app without min authing again. Need to discuss further for next steps forward.
